<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSI Motion Detection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QWidget" name="topWidget">
       <layout class="QVBoxLayout" name="mainLayout">
        <item>
         <widget class="QGroupBox" name="plotGroupBox">
          <property name="title">
           <string>CSI Spectrogram</string>
          </property>
          <layout class="QVBoxLayout" name="plot_layout"/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="controlsLayout">
          <item>
           <widget class="QGroupBox" name="detectionGroupBox">
            <property name="title">
             <string>Detection</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="thresholdLabel">
               <property name="text">
                <string>Threshold (disabled or 1000-3000):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="thresholdSlider">
               <property name="minimum">
                <number>1000</number>
               </property>
               <property name="maximum">
                <number>3000</number>
               </property>
               <property name="value">
                <number>1800</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
               <property name="tickInterval">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="thresholdValueLabel">
               <property name="text">
                <string>50</string>
               </property>
               <property name="minimumWidth">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QCheckBox" name="defaultThresholdCheckBox">
               <property name="text">
                <string>Enable Detection</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="4">
              <widget class="QLineEdit" name="alertLineEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>No motion detected</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font-weight: bold; color: #d9534f;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonLayout">
            <item>
             <widget class="QPushButton" name="startButton">
              <property name="text">
               <string>Start</string>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #5cb85c; color: white;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopButton">
              <property name="text">
               <string>Stop</string>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #d9534f; color: white;</string>
              </property>
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="logGroupBox">
       <property name="title">
        <string>Event Log</string>
       </property>
       <property name="maximumHeight">
        <number>150</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextEdit" name="logText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="maximumHeight">
           <number>120</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
</ui>